### PR TITLE
update user doc for multirocksdb

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: partitioned-raft-kv.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
...
-* 暂不支持 Online Unsafe Recovery 和 Titan。
-* 暂不支持 tikv-ctl 命令行管理工具。
-* 初始化以后不支持开启或者关闭。
+* 暂不支持基于 EBS 的快照备份
+* 暂不支持 Online Unsafe Recovery 和 Titan
+* tikv-ctl命令行管理工具中以下子命令不再支持
+  * unsafe-recover
+  * raw-scan
+* 初始化以后不支持开启或者关闭
+* 暂不支持和TiFlash的兼容
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:44]:
```suggestion
* 暂不支持基于 EBS 的快照备份
* 暂不支持 Online Unsafe Recovery 和 Titan
* 不支持 tikv-ctl 命令行管理工具中的以下子命令：
    * `unsafe-recover`
    * `raw-scan`
* 初始化以后不支持开启或者关闭 Partitioned Raft KV
* 暂不支持和 TiFlash 的兼容
```

Partitioned Raft KV 在 [7.1.0 release notes](https://docs.pingcap.com/zh/tidb/stable/release-7.1.0#%E5%8A%9F%E8%83%BD%E8%AF%A6%E6%83%85) 中已经说明了“**在 TiDB v7.1.0 中，Partitioned Raft KV 引擎与 TiFlash 兼容**”。
@tonyxuqqi 请确认 Partitioned Raft KV “暂不支持和 TiFlash 的兼容”是指 v7.1.0 支持，但是从 v7.3.0 开始不支持吗？

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1419,7 +1419,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 + memtable 大小。
 + `defaultcf` 默认值：`"128MB"`
 + `writecf` 默认值：`"128MB"`
-+ `lockcf` 默认值：`"32MB"`
++ `lockcf` 默认值：当storage.engine为raft-kv时为`"32MB"`; 当storage.engine为partitioned-raft-kv时为`"4MB"`
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1422]:
```suggestion
+ `lockcf` 默认值：
    + 当 `storage.engine="raft-kv"` 时，默认值为 `"32MB"`
    + 当 `storage.engine="partitioned-raft-kv"` 时，默认值为 `"4MB"`
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1707,7 +1707,7 @@ raftdb 相关配置项。
 ### `max-total-wal-size`
 
 + RocksDB WAL 文件的最大总大小。
-+ 默认值：`"4GB"`
++ 默认值：当storage.engine为raft-kv时默认值为`"4GB"`; 当storage.engine为raft-kv时默认值为1（相当于禁用)
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1710]:
```suggestion
+ 默认值：
    + 当 `storage.engine="raft-kv"` 时，默认值为 `"4GB"`
    + 当 `storage.engine="partitioned-raft-kv"` 时，默认值为 `1`，相当于没有限制
```
”禁用”是指不限制 WAL 文件大小吗

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1862,7 +1862,7 @@ Raft Engine 相关的配置项。
 + 可选值：
     + `1`：v6.3.0 以前的默认日志文件格式。v6.1.0 及以后版本的 TiKV 可以读取该格式。
     + `2`：支持日志回收。v6.3.0 及以后版本的 TiKV 可以读取该格式。
-+ 默认值：`2`
++ 默认值：当storage.engine为raft-kv时为`2`;当storage.engine为partitioned-raft-kv时为`5`
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1865]:
```suggestion
+ 默认值：
    + 当 `storage.engine="raft-kv"` 时，默认值为 `2`
    + 当 `storage.engine="partitioned-raft-kv"` 时，默认值为 `5`
```

File: partitioned-raft-kv.md (hunk 1 above)
Comment (tonyxuqqi) [reply partitioned-raft-kv.md:44]: 7.1开始就不支持吧，可能代码上试图支持了，但测试没跑完。

File: tikv-configuration-file.md (hunk 3 above)
Comment (tonyxuqqi) [reply tikv-configuration-file.md:1712]: 指的是禁用WAL（因为WAL只能有一个字节）

File: partitioned-raft-kv.md (hunk 1 above)
Comment (Oreoxmt) [inline partitioned-raft-kv.md:44]:
```suggestion
* 暂不支持基于 EBS 的快照备份
* 暂不支持 Online Unsafe Recovery 和 Titan
* 不支持 tikv-ctl 命令行管理工具中的以下子命令：
    * `unsafe-recover`
    * `raw-scan`
* 初始化以后不支持开启或者关闭 Partitioned Raft KV
* 暂不支持和 TiFlash 的兼容
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1707,7 +1709,7 @@ raftdb 相关配置项。
 ### `max-total-wal-size`
 
 + RocksDB WAL 文件的最大总大小。
-+ 默认值：`"4GB"`
++ 默认值：当storage.engine为raft-kv时默认值为`"4GB"`; 当storage.engine为raft-kv时默认值为1（相当于禁用)
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1712]:
```suggestion
+ 默认值：
    + 当 `storage.engine="raft-kv"` 时，默认值为 `"4GB"`
    + 当 `storage.engine="partitioned-raft-kv"` 时，默认值为 `1`，相当于禁用 WAL 文件
```

**Review by hfxsd — APPROVED** [review]

File: partitioned-raft-kv.md [hunk 6]
```diff
@@ -37,6 +37,8 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 * 暂不支持基于 EBS 的快照备份
 * 暂不支持 Online Unsafe Recovery 和 Titan
-* 暂不支持 tikv-ctl 命令行管理工具
+* 不支持 tikv-ctl 命令行管理工具中的以下子命令：
```
Comment (SpadeA-Tang) [inline partitioned-raft-kv.md:40]: the unsafe opeartions that tikv-ctl v2 does not support includes: remove-fail-stores, recreate-region, and reset-to-version.

**Review by Oreoxmt — APPROVED** [review]

**Review by SpadeA-Tang — APPROVED** [review]

**Pull request merged**